module-services/service-cellular/ServiceCellular.cpp

```diff
@@ -1371,7 +1371,7 @@ bool ServiceCellular::handle_URCReady()
     }
 
     bool ret = true;
-    ret = ret && channel->cmd(at::AT::ENABLE_NETWORK_REGISTRATION_URC);
+    ret      = ret && channel->cmd(at::AT::ENABLE_NETWORK_REGISTRATION_URC);
```

Comment:
wouldn't  `bool ret = channel->cmd(at::AT::ENABLE_NETWORK_REGISTRATION_URC);` do the same thing?